nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/deck_guns.dm

```diff
@@ -432,6 +432,10 @@
 		if(OM.z == z)
 			shake_with_inertia(M, 1, 1)
 
+/obj/machinery/ship_weapon/deck_turret/overmap_fire(atom/target)
+	linked.shake_animation()
+	. = ..()
```

Comment:
I *believe* it's (barely) more performant to return instead of proc var setting parent at the end of a proc. Plus it's neater too :)
```suggestion
	return ..()
```